test/modules/authorizer/SingleVoteGovernor.t.sol

```diff
@@ -224,13 +224,13 @@ contract SingleVoteGovernorTest is ModuleTest {
     function testInit() public override(ModuleTest) {
         assertEq(_orchestrator.isModule(address(_governor)), true);
 
-        assertEq(_authorizer.isAuthorized(address(_governor)), true);
+        assertEq(_authorizer.hasRole("0x01", address(_governor)), true); // Owner role
```

Comment:
Direclty using role 0x01 is pretty hard to understand without any prior knowledge, maybe you can instead reference the authorizer versions directly (authorizer.OWNER())